rocketpy/Function.py

```diff
@@ -5,9 +5,16 @@
 __license__ = "MIT"
 
 from inspect import signature
+from copy import copy
+
+try:
+    from functools import cached_property
+except ImportError:
+    from .tools import cached_property
 
 import matplotlib.pyplot as plt
 import numpy as np
+from functools import cached_property
 from scipy import integrate, linalg, optimize
```

Comment:
I believe this is the line that is causing the error in the tests

I suggest to just remove it:

```suggestion
from scipy import integrate, linalg, optimize
```